js/tablenator.js

```diff
@@ -1,6 +1,9 @@
 namespacer('bcpl');
 
 bcpl.tablenator = (($, _) => {
+	const responsiveTableClass = 'tablenator-responsive-table';
+	const tablenatorPrefix = 'tablenator-';
```

Comment:
Should `tablenatorPrefix` be used as part of `responsiveTableClass`, in case you want to change the prefix?

Reply:
yea i was torn on that too, i need to use the prefix to be consistentin the code, but i debated letting you pass in the container class as an option. This is really a finished thought, ill make a note and come back to it.